packages/neon/neon/lib/src/utils/global_popups.dart

```diff
@@ -31,27 +32,31 @@ class GlobalPopups {
     final globalOptions = Provider.of<GlobalOptions>(context, listen: false);
     final firstLaunchBloc = Provider.of<FirstLaunchBloc>(context, listen: false);
     final nextPushBloc = Provider.of<NextPushBloc>(context, listen: false);
+    final platform = Provider.of<NeonPlatform>(context, listen: false);
 
     firstLaunchBloc.onFirstLaunch.listen((final _) {
-      if (globalOptions.pushNotificationsEnabled.enabled) {
-        if (!context.mounted) {
```

Comment:
Are we sure that the context is always mounted when the the listener gets called?

Reply:
The home page is always kept alive once it has been created, right? Then the context is always mounted.

Reply:
Than we should add an assert.

While this is true for now I'd like to prevent this popping up in the future

Reply:
I implemented the dispose logic now, but then it's not even needed, right? Or shall we just keep it so we don't forget it in the future if things may change?

Reply:
I think we should keep it.